webpack/components/extensions/HostDetails/Cards/ContentViewDetailsCard.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  Flex,
+  FlexItem,
+  GridItem,
+  Label,
+} from '@patternfly/react-core';
+
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { propsToCamelCase } from 'foremanReact/common/helpers';
+import PropTypes from 'prop-types';
+import ContentViewIcon from '../../../../scenes/ContentViews/components/ContentViewIcon';
+
+const renderContentViewDetails = (hostDetails) => {
```

Comment:
This is a function that returns JSX, therefore I think this is really another React component but named like a regular function. Let's fix that:
```suggestion
const HostContentViewDetails = (hostDetails) => {
```